src/archanaNJun20/PalindromeNumber.java

```diff
@@ -0,0 +1,22 @@
+package archanaNJun20;
+
+public class PalindromeNumber {
+	int rev=0;
```

Comment:
Instead of defining "rev" as instance variable you can define it in method itself as a local variable.

Reply:
The code is updated based on the comment. Thank you for your Suggestion.